Packaging/amiga/prep.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+#set compiler params
+export TARGET='m68k-amigaos'
+export SYSROOT=/opt/$TARGET
+export FLAGS='-m68040 -mhard-float -s -ffast-math -O3 -w -fomit-frame-pointer  -noixemul -DBIG_ENDIAN -DAMIGA -fpermissive -std=c++14'
+export DEFINES='-DM68K_CPU=68040 -DM68K_FPU=hard -DM68K_COMMON="-s -ffast-math -O3 -fno-exceptions -w -fomit-frame-pointer  -noixemul -DBIG_ENDIAN -DAMIGA -fpermissive -std=c++14"'
+
+# ZLIB
+wget https://www.zlib.net/zlib-1.2.11.tar.gz
+tar -xvf zlib-1.2.11.tar.gz
+mkdir -p zlib-1.2.11/build
+cd zlib-1.2.11/build
+eval cmake .. -DCMAKE_INSTALL_PREFIX=${SYSROOT} ${DEFINES}
+cmake --build . --config Release --target install -- -j$(nproc)
```

Comment:
Use `getconf _NPROCESSORS_ONLN` instead of `nproc` for POSIX compatibility (MacOS, msys2, etc)

Reply:
Fixed